src/features/PurchaseMembership/components/MembershipSelector.tsx

```diff
@@ -68,7 +66,7 @@ export default function MembershipSelector({ name, value, onSelect }: Membership
       options={options}
       onChange={(option) => onSelect(option)}
       getOptionLabel={(option) => option.title}
-      optionComponent={({ data, selectOption }: OptionProps<IMembershipConfig>) => (
+      optionComponent={({ data, selectOption }: OptionProps<IMembershipConfig, false>) => (
```

Comment:
What is the extra flag here for?

Reply:
This is the `isMulti` flag. It's to define if the selector allow multiple selections